spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -147,6 +148,15 @@ public void saveEntities() {
 
 	}
 
+	@Test
+	public void writeDeleteTest() {
+		List<EmbeddableTreeNode> nodes = IntStream.rangeClosed(1, 600)
```

Comment:
I hope this test doesn't take too long. How long does it take?

Reply:
It takes about 12.5 seconds